addons/weaponselect/functions/fnc_throwGrenade.sqf

```diff
@@ -1,10 +1,10 @@
 // by commy2
 #include "script_component.hpp"
 
-private ["_unit", "_weapon", "_magazine"];
+private ["_magazine"];
+
+PARAMS_2(_unit,_weapon);
 
-_unit = _this select 0;
-_weapon = _this select 1;
 _magazine = _this select 5;
```

Comment:
I agree, we shouldn't use PARAMS_X or EXPLODE_X_PVT if not all the variables are extracted.